src/solvers/strings/string_builtin_function.cpp

```diff
@@ -182,7 +182,7 @@ optionalt<exprt> string_concat_char_builtin_functiont::eval(
   }();
   input_opt.value().push_back(char_val);
   const auto length =
-    from_integer(input_opt.value().size(), result.length().type());
+    from_integer(input_opt.value().size(), result.length_type());
```

Comment:
:pick: In the commit message: "everywhere" is too imprecise

Reply:
Added a commit description